packages/models/proto/v2/eservice/eservice.proto

```diff
@@ -0,0 +1,78 @@
+syntax = "proto3";
+
+package eservice.v2;
+
+message EServiceV2 {
+  string id = 1;
+  string producerId = 2;
+  string name = 3;
+  string description = 4;
+  EServiceTechnologyV2 technology = 5;
+  repeated EServiceDescriptorV2 descriptors = 6;
+  int64 createdAt = 7;
+}
+
+message EServiceAttributeValueV2 {
+  string id = 1;
+  bool explicitAttributeVerification = 2;
+}
+
+message EServiceAttributeV2 {
+  optional EServiceAttributeValueV2 single = 1;
+  repeated EServiceAttributeValueV2 group = 2;
```

Comment:
now that there is no difference we can generalize the name
```suggestion
  repeated EServiceAttributeValueV2 values = 1;
```

Reply:
At this poitn isn't better to directly use the `EServiceAttributeValueV2` message and eliminate the current `EServiceAttributeV2` message (maybe renaming `EServiceAttributeValueV2` into `EServiceAttributeV2`)

```
message EServiceAttributeV2 {
  string id = 1;
  bool explicitAttributeVerification = 2;
}

message EServiceAttributesV2 {
  repeated EServiceAttributeV2 certified = 1;
  repeated EServiceAttributeV2 declared = 2;
  repeated EServiceAttributeV2 verified = 3;
}

message EServiceDescriptorV2 {
  string id = 1;
  int64 version = 2;
  optional string description = 3;
  repeated EServiceDocumentV2 docs = 4;
  EServiceDescriptorStateV2 state = 5;
  optional EServiceDocumentV2 interface = 6;
  repeated string audience = 7;
  int32 voucherLifespan = 8;
  int32 dailyCallsPerConsumer = 9;
  int32 dailyCallsTotal = 10;
  AgreementApprovalPolicyV2 agreementApprovalPolicy = 11;
  int64 createdAt = 12;
  optional int64 publishedAt = 13;
  repeated string serverUrls = 14;
  optional int64 suspendedAt = 15;
  optional int64 deprecatedAt = 16;
  optional int64 archivedAt = 17;
  EServiceAttributesV2 attributes = 18;
}
```

Reply:
`certified` (and the others) should be a `List[List[_]]`, which is why the extra type.
But if there is a different way to model this it's ok for me!